changelogs/fragments/109-eos-ospfv3-resource-module.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```

Comment:
Should this be a minor change? I'm not sure if `major_changes` would entail a major version bump.